enhancements/custom-alert-configuration.md

```diff
@@ -0,0 +1,438 @@
+---
+title: custom-alert-configuration
+authors:
+  - "@atiratree"
+reviewers:
+  - "@deads2k"
+  - "@soltysh"
+  - "@fabiand"
+  - "@acardace"
+approvers:
+  - "@deads2k"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-01-26
+last-updated: 2023-01-26
+tracking-link:
+  - "https://issues.redhat.com/browse/CNV-15064"
+see-also:
+  - "https://github.com/openshift/cluster-kube-controller-manager-operator/pull/583"
+---
+
+
+# Custom Alert Configuration
+
+## Summary
+
+There is a need for a mechanism for a certain use cases, that would allow
+OpenShift components to opt out kubernetes objects from being picked up by
+alerts. The impact of missing this feature is felt most by users of KubeVirt
+(OpenShift Virtualization). This feature should not be available to users
+because it could disrupt existing alerting/monitoring. Users should use
+alert silences instead.
+
+## Motivation
+
+Running virtual machine consists of 3 objects VirtualMachine,
+VirtualMachineInstance and a pod that is running the VM (virt-launcher-).
+KubeVirt has a [Live Migration][1] feature that can move this pod which is running
```

Comment:
`KubeVirt has a [Live Migration][1] feature that uses a two pod mechanism to move`